lib/produce/config.rb

```diff
@@ -84,5 +86,9 @@ def is_valid_language? language
     def skip_itc? value
       %w( true t 1 yes y ).include? value.to_s.downcase
     end
+
+    def skip_devcenter? value
```

Comment:
You're right! I also added a small note in the code as a remainder to put that method inside fastlane_core